ui/desktop/package.json

```
@@ -14,7 +14,7 @@
     "build": "yarn build:production && yarn build:desktop",
     "build:production": "cross-env EMBER_CLI_ELECTRON=true ember build --environment=production",
     "build:development": "ember build",
-    "prebuild:desktop": "shx rm -rf electron-app/ember-dist && shx cp -r dist/ electron-app/ember-dist/",
+    "prebuild:desktop": "shx rm -rf electron-app/ember-dist && shx cp -r dist/ electron-app/ember-dist/ && shx rm -rf electron-app/out",
```

Could you have just included it as part of the first `rm -rf`? 

```suggestion
    "prebuild:desktop": "shx rm -rf electron-app/{ember-dist,out} && shx cp -r dist/ electron-app/ember-dist/",
```

or `rm -rf electron-app/ember-dist electron-app/out` if brace expansion doesn't work with shx (not sure)

---

Yes! thanks for the suggeston.

Brace expansion didn't work on windows, so opted for just chaining.